python/setup.py

```diff
@@ -586,7 +586,7 @@ def parse_git(root, **kwargs):
     """
     from setuptools_scm.git import parse
     kwargs['describe_command'] =\
-        'git describe --dirty --tags --long --match "apache-arrow-[0-9].*"'
+        'git describe --dirty --tags --long --match "apache-arrow-[0-9]*.*"'
```

Comment:
Wow, how many instances of this do we have? :-)
```suggestion
        'git describe --dirty --tags --long --match "apache-arrow-[0-9]+.*"'
```